README.md

```diff
@@ -100,12 +100,17 @@ Parameters:
 
  * `source`: The name of the cookbook file (required)
  * `cookbook`: The name of the cookbook to find the file (optional). Defaults to the cookbook calling the LWRP
+ * `mode`: The permissions to set on the cookbook file (optional). Defaults to '750'
+ * `only_if`: Use a chef only_if [guard](https://docs.chef.io/resource_common.html#guards] to only write the cookbook file if the statement evaluates to true (optional).
+ * `not_if`: Use a chef not_if [guard](https://docs.chef.io/resource_common.html#guards] to not write the cookbook file if the statement evaluates to true (optional).
 
 Example:
 ``` ruby
 cerner_tomcat "my_tomcat" do
   cookbook_file "lib/mysql.jar" do
     source "mysql-5.12.jar"
+    only_if { true }
+    not_if { false }
```

Comment:
Mind putting a slightly more complex example in here? Maybe something like File.exists?('mysql-5.12.jar')?